src/components/Search/index.jsx

```diff
@@ -0,0 +1,70 @@
+import algoliasearch from 'algoliasearch/lite';
+import { default as React, useState, useMemo } from 'react';
+import { InstantSearch } from 'react-instantsearch-dom';
+import { makeStyles, Box, List, ClickAwayListener } from '@material-ui/core';
+import SearchBox from './search-box';
+import SearchResult from './search-result';
+
+const useStyles = makeStyles((classes) => ({
+  resultsOpen: {
+    background: 'white',
+    position: 'relative',
+  },
+  resultsBorder: {
+    border: `1px solid ${classes.palette.divider}`,
+    maxHeight: 580,
+    overflow: 'auto',
+    overflowX: 'hidden',
+  },
```

Comment:
This border can be deleted now that the results are inline.

```suggestion
```

And then the corresponding `<Box>` can go away too.